### PR TITLE
Package definition view styles updates

### PR DESCRIPTION
**Pull request by roonyh**

### CODE REVIEW

File: modules/web/js/ballerina/components/package-definition.jsx [hunk 1]
```diff
@@ -92,14 +95,21 @@ class PackageDefinition extends React.Component {
 
         return (
             <g>
-                <rect x={ bBox.x } y={ bBox.y } width={ bBox.w } height={ headerHeight } rx="0" ry="0" className="package-definition-header"/>
+                <rect x={ bBox.x } y={ bBox.y } width={310} height={ headerHeight } rx="0" ry="0" className="package-definition-header"/>
```
Comment (nadeeshaan) [inline modules/web/js/ballerina/components/package-definition.jsx]: can't we either calculate this value of 310, or get it from constants?

File: modules/web/js/ballerina/visitors/position-calc/package-def.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+ * under the License.
+ */
+
+import log from 'log';
+import * as Utils from './utils';
+
+class PackageDefinitionPositionCalcVisitor {
+
+    canVisit(node) {
+        log.debug('can visit FunctionInvocationStatementPositionCalc');
```
Comment (nadeeshaan) [inline modules/web/js/ballerina/visitors/position-calc/package-def.js]: change the debug message. FunctionInvocationStatementPositionCalc to package definition

File: modules/web/js/ballerina/visitors/position-calc/package-def.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+        return true;
+    }
+
+    beginVisit(node) {
+        log.debug('begin visit FunctionInvocationStatementPositionCalc');
```
Comment (nadeeshaan) [inline modules/web/js/ballerina/visitors/position-calc/package-def.js]: change the debug message. FunctionInvocationStatementPositionCalc to package definition

File: modules/web/js/ballerina/visitors/position-calc/package-def.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+        let viewSate = node.getViewState();
+        let bBox = viewSate.bBox;
+
+        bBox.x = 50;
+        bBox.y = 20;
+    }
+
+    visit(node) {
+        log.debug('visit FunctionInvocationStatementPositionCalc');
```
Comment (nadeeshaan) [inline modules/web/js/ballerina/visitors/position-calc/package-def.js]: change the debug message. FunctionInvocationStatementPositionCalc to package definition

File: modules/web/js/ballerina/visitors/position-calc/package-def.js [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+    }
+
+    endVisit(node) {
+        log.debug('end visit FunctionInvocationStatementPositionCalc');
```
Comment (nadeeshaan) [inline modules/web/js/ballerina/visitors/position-calc/package-def.js]: change the debug message. FunctionInvocationStatementPositionCalc to package definition

File: modules/web/js/ballerina/visitors/position-calc/package-def.js [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import log from 'log';
+import * as Utils from './utils';
+
+class PackageDefinitionPositionCalcVisitor {
```
Comment (hemikak) [inline modules/web/js/ballerina/visitors/position-calc/package-def.js]: Can we have a class comment ?

File: modules/web/js/ballerina/components/package-definition.jsx (hunk 1 above)
Comment (roonyh) [inline modules/web/js/ballerina/components/package-definition.jsx]: done in #2006

File: modules/web/js/ballerina/visitors/position-calc/package-def.js (hunk 2 above)
Comment (roonyh) [inline modules/web/js/ballerina/visitors/position-calc/package-def.js]: Changed in #2010. Thanks for the catch! :)

File: modules/web/js/ballerina/visitors/position-calc/package-def.js (hunk 3 above)
Comment (roonyh) [inline modules/web/js/ballerina/visitors/position-calc/package-def.js]: Changed in #2010

File: modules/web/js/ballerina/visitors/position-calc/package-def.js (hunk 4 above)
Comment (roonyh) [inline modules/web/js/ballerina/visitors/position-calc/package-def.js]: Changed in #2010

File: modules/web/js/ballerina/visitors/position-calc/package-def.js (hunk 5 above)
Comment (roonyh) [inline modules/web/js/ballerina/visitors/position-calc/package-def.js]: Changed in #2010

File: modules/web/js/ballerina/visitors/position-calc/package-def.js (hunk 6 above)
Comment (roonyh) [inline modules/web/js/ballerina/visitors/position-calc/package-def.js]: This is already added during the linting effort

**Pull request merged**